### PR TITLE
update novelupdates & lightnovelpub & kolnovel

### PR DESCRIPTION
**Pull request by K1ngfish3r**
add lightnovelstranslations to chapterText check
add inoveltranslation to chapterText check (fixes https://github.com/LNReader/lnreader-sources/issues/483)

### CODE REVIEW

**Comment by K1ngfish3r** [comment]
fixes https://github.com/LNReader/lnreader-sources/issues/494

**Comment by K1ngfish3r** [comment]
Add a search to lightnovelpub

**Comment by K1ngfish3r** [comment]
added a fix (maybe) for KoLnovel's AI text
https://github.com/LNReader/lnreader-sources/issues/504
https://github.com/LNReader/lnreader-sources/issues/508

cant confirm since I cant read Arabic but it's consistent in 3 novels i've tested.

**Comment by K1ngfish3r** [comment]
refactor indowebnovel (fixes https://github.com/LNReader/lnreader-sources/issues/470)
remove adsbygoogle message in sakuranovel description

File: src/sources/multisrc/wqmangastream/WQMangaStreamScraper.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ class WQMangaStreamScraper {
     this.sourceName = sourceName;
     this.language = options?.language;
     this.reverseChapters = options?.reverseChapters;
+    this.AI_Text = options?.AI_Text;
```
Comment (rajarsheechatterjee) [inline src/sources/multisrc/wqmangastream/WQMangaStreamScraper.js:11]: To be generic this should be something like `ignoredStrings` which would be an array of strings to filter out.

**Comment by K1ngfish3r** [comment]
my pr not updating, i go delete my repo and make new pr

**Pull request closed without merging**